applications/asset_tracker/configuration/nrf9160_pca20035ns/pm_static.yml

```diff
@@ -0,0 +1,53 @@
+app: {address: 0x18200, size: 0x5ae00}
+mcuboot:
+  address: 0x0
+  placement:
+    before: [mcuboot_primary]
+  size: 0xc000
+mcuboot_pad:
+  address: 0xc000
+  placement:
+    align: {start: 0x1000}
+    before: [mcuboot_primary_app]
+  size: 0x200
+mcuboot_primary:
+  address: 0xc000
+  orig_span: &id001 [spm, mcuboot_pad, app]
+  sharers: 0x1
+  size: 0x69000
+  span: *id001
+mcuboot_primary_app:
+  address: 0xc200
+  orig_span: &id002 [app, spm]
+  size: 0x68e00
+  span: *id002
+mcuboot_scratch:
+  address: 0xde000
+  placement:
+    after: [app]
+    align: {start: 0x1000}
+  size: 0x1e000
+mcuboot_secondary:
+  address: 0x75000
+  placement:
+    after: [mcuboot_primary]
+    align: {start: 0x1000}
+  share_size: [mcuboot_primary]
+  size: 0x69000
+mcuboot_storage:
```

Comment:
I assume you are not using this in your application?
I'm aligning it more a bit towards how `nrf_desktop` have their configuration by putting settings in the top of application processor flash.

The alignment will look like this for thingy now:
```
+-----------+
|  mcuboot  |
|-----------|
|  primary  |
|-----------|
| secondary |
|-----------|
|  scratch  |
|-----------|
|  padding  |
|-----------|
| settings  |
+-----------+

Reply:
I assume you are not fixed on keeping the same configuration for pca10090? Since it already has the SEGGER chip on it.

Reply:
Tested with v0.2.2 and it seemed to work with FOTA at least.

Reply:
Looks like a good approach!